jd4/case.py

```diff
@@ -25,6 +25,7 @@
 DEFAULT_TIME_NS = 1000000000
 DEFAULT_MEMORY_BYTES = 268435456
 PROCESS_LIMIT = 64
+RE_INPUT_FILENAME = re.compile(r'([a-zA-Z]*)([0-9]+)\.in')
```

Comment:
Problem may not have an input file, but have an output file.

Reply:
> Set .out back to .in as there won't be .out files in custom judge case.

What about use `(in|out)` here, and use a dict to store if `match.group(2)` appeared before?

@twd2 PTAL

Reply:
Such Soha so smart

Reply:
I'm not sure if a program can run properly without stdin linked to an input file.

Reply:
Actually I don't think it necessary to support this
And looking up the dict will influence the performance a little bit.

Reply:
Using `(in|out)` seems more correct, since we are planning to support such cases without stdin file.

Reply:
![image](https://user-images.githubusercontent.com/29992205/56091579-4897b600-5ee3-11e9-9ac7-c93950473330.png)
This might raise an error if file doesn't exist.

Reply:
OK, we can change to `(in|out)` later.

Reply:
Opened #72.